edit.js

```diff
@@ -11,39 +11,73 @@ appliesToEverythingTemplate.innerHTML = t("appliesToEverything") + ' <button cla
 var sectionTemplate = document.createElement("div");
 sectionTemplate.innerHTML = '<label>' + t('sectionCode') + '</label><textarea class="code"></textarea><br><div class="applies-to"><label>' + t("appliesLabel") + ' <img class="applies-to-help" src="help.png" alt="' + t('helpAlt') + '"></label><ul class="applies-to-list"></ul></div><button class="remove-section">' + t('sectionRemove') + '</button><button class="add-section">' + t('sectionAdd') + '</button>';
 
-
 var editors = [] // array of all CodeMirror instances
+function initCodeMirror() {
+	var CM = CodeMirror,
+	    dd = CM.defaults,
+		cc = CM.commands;
+	// default option values
+	dd.mode = 'css';
+	dd.lineNumbers = true;
+	dd.lineWrapping = true;
+	dd.foldGutter = true;
+	dd.gutters = ["CodeMirror-linenumbers", "CodeMirror-foldgutter", "CodeMirror-lint-markers"];
+	dd.matchBrackets = true;
+	dd.lint = CodeMirror.lint.css;
+	dd.smartIndent = prefs.getPref("smart-indent", true);
+	dd.keyMap = "sublime";
+	dd.extraKeys = {"Ctrl-Space": "autocomplete"};
+
+	// additional commands
+	cc.jumpToLine = jumpToLine;
+	cc.nextBuffer = nextBuffer;
+	cc.prevBuffer = prevBuffer;
+	cc.save = save;
```

Comment:
`cc.save = save;` should be removed, see https://github.com/JasonBarnabe/stylish-chrome/commit/0be6f6dfe440339a7e20f5d203722ee0deaab2cd and https://github.com/JasonBarnabe/stylish-chrome/commit/464d3eb30db9b28fb17f22b89bdcc16472b133e4

Also [L18](https://github.com/JasonBarnabe/stylish-chrome/pull/64/files#diff-3cc18e14de6e30bc468459dbeb194f7dR18) above has two tabs indent.